.readthedocs.yaml

```diff
@@ -0,0 +1,22 @@
+# Readthedocs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Readthedocs configuration file version
+version: 2
+
+# Additional document formats to produce
+formats: all
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.7"
```

Comment:
This choice of Python 3.7 is somewhat arbitrary, but at least it matches the previous Python version used before the explicit `.readthedocs.yaml` config file was used. See https://readthedocs.org/projects/simphony-remote/builds/15553588/#build-commands